src/schema/v2/user.ts

```diff
@@ -79,6 +117,28 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
       description: "The given phone number of the user.",
       type: GraphQLString,
     },
+    createdAt: date(({ created_at }) => created_at),
+    emailConfirmedAt: date(({ email_confirmed_at }) => email_confirmed_at),
+    secondFactorEnabled: {
+      description:
+        "If the user has enabled two-factor authentication on their account",
+      type: GraphQLBoolean,
```

Comment:
Generally wondering about nullability of these fields!

Reply:
Good calling-out! Since I'm adding a fair amount of additional fields and loaders, I focused on getting all the data returned properly first, but still need to go through and confirm which are nullable and which aren't.